src/simple-todos-graphql/step01/client/main.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Meteor } from 'meteor/meteor';
+import { render } from 'react-dom';
+import { App } from '/imports/ui/App';
+
+import { startGraphQLClient } from 'meteor/quave:graphql/client';
```

Comment:
As much as `quave:graphql` is fine, I think we should stick to the official one via skeleton.